web/src/main/java/model/Transaction.java

```diff
@@ -0,0 +1,43 @@
+package model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import model.enumerations.Type;
+
+import java.util.Date;
+
+@Data
+@Entity
+public class Transaction {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private Date date;
+
+    @ManyToOne
+    private Account account;
+
+    private String description;
+
+    private double amount;
```

Comment:
The amount should be of type Money. Create a new class (which we won't persist) with two fields: BigDecimal value and Enum currency. Create enum currency, where we will have control over which currencies we support.